src/components/ZestyExplorer.tsx

```diff
@@ -197,10 +197,52 @@ function canUseDOM() {
   );
 }
 
-export const ZestyExplorer = ({ content }: any) => {
-  console.log('***********ZESTY EXPLORER LOADED *************');
+const getPageData = async () => {
+  let data = {
+    error: true,
+    production: true,
+  };
+  const queryString = window.location.search.substring(1);
+
+  const prodUri = window.location.href + '?toJSON&' + queryString;
+  const devUri =
```

Comment:
This is not needed, URI will already be the URL from the browser, you do not need logic to change it. There will need to be logic to turn it off or on per the environment.